src/rdkafka.h

```diff
@@ -5249,7 +5249,7 @@ typedef rd_kafka_resp_err_t
  * @param errstr A human readable error string in case the interceptor fails.
  * @param errstr_size Maximum space (including \0) in \p errstr.
  *
- * @returns RD_KAFKA_CONF_RES_OK if the property was known and successfully
+ * @returns RD_KAFKA_CONF_OK if the property was known and successfully
  *          handled by the interceptor, RD_KAFKA_CONF_RES_INVALID if the
```

Comment:
can you fix these two as well? RES_INVALID and RES_UNKNOWN?

Reply:
Sure, this is just something i noticed while reading through the docs